include/z64save.h

```diff
@@ -578,47 +580,47 @@ typedef enum LinkAge {
-#define EVENTCHKINF_A0 0xA0
-#define EVENTCHKINF_A1 0xA1
-#define EVENTCHKINF_A3 0xA3
-#define EVENTCHKINF_A4 0xA4
-#define EVENTCHKINF_A5 0xA5
-#define EVENTCHKINF_A6 0xA6
-#define EVENTCHKINF_A7 0xA7
-#define EVENTCHKINF_A8 0xA8
-#define EVENTCHKINF_A9 0xA9
-#define EVENTCHKINF_AA 0xAA
-#define EVENTCHKINF_AC 0xAC
-#define EVENTCHKINF_AD 0xAD
-#define EVENTCHKINF_B0 0xB0
-#define EVENTCHKINF_B1 0xB1
-#define EVENTCHKINF_B2 0xB2
-#define EVENTCHKINF_B3 0xB3
-#define EVENTCHKINF_B4 0xB4
-#define EVENTCHKINF_B5 0xB5
-#define EVENTCHKINF_B6 0xB6
-#define EVENTCHKINF_B7 0xB7
-#define EVENTCHKINF_B8 0xB8
-#define EVENTCHKINF_B9 0xB9
-#define EVENTCHKINF_BA 0xBA
-#define EVENTCHKINF_BB 0xBB
-#define EVENTCHKINF_BC 0xBC
-#define EVENTCHKINF_BD 0xBD
-#define EVENTCHKINF_BE 0xBE
-#define EVENTCHKINF_BF 0xBF
-#define EVENTCHKINF_C0 0xC0
-#define EVENTCHKINF_C1 0xC1
-#define EVENTCHKINF_C3 0xC3
-#define EVENTCHKINF_C4 0xC4
-#define EVENTCHKINF_C5 0xC5
-#define EVENTCHKINF_C6 0xC6
-#define EVENTCHKINF_C7 0xC7
-#define EVENTCHKINF_C8 0xC8
-#define EVENTCHKINF_C9 0xC9
+#define EVENTCHKINF_SPOKE_TO_NABOORU_IN_SPIRIT_TEMPLE 0x94
+#define EVENTCHKINF_NABOORU_CAPTURED_BY_TWINROVA 0x95
+#define EVENTCHKINF_SPOKE_TO_CURSED_MAN_IN_SKULL_HOUSE 0x96
+#define EVENTCHKINF_PLAYED_SONG_FOR_SCARECROW_AS_ADULT 0x9C
+#define EVENTCHKINF_ENTERED_HYRULE_FIELD 0xA0
+#define EVENTCHKINF_ENTERED_DEATH_MOUNTAIN_TRAIL 0xA1
+#define EVENTCHKINF_ENTERED_KAKARIKO_VILLAGE 0xA3
+#define EVENTCHKINF_ENTERED_ZORAS_DOMAIN 0xA4
+#define EVENTCHKINF_ENTERED_HYRULE_CASTLE 0xA5
+#define EVENTCHKINF_ENTERED_GORON_CITY 0xA6
+#define EVENTCHKINF_ENTERED_TEMPLE_OF_TIME 0xA7
+#define EVENTCHKINF_ENTERED_DEKU_TREE 0xA8
+#define EVENTCHKINF_LEARNED_SONG_OF_TIME 0xA9
+#define EVENTCHKINF_BONGO_BONGO_ESCAPED_FROM_WELL 0xAA
+#define EVENTCHKINF_LEARNED_REQUIEM_OF_SPIRIT 0xAC
+#define EVENTCHKINF_COMPLETED_SPIRIT_TRIAL 0xAD
+#define EVENTCHKINF_ENTERED_DODONGOS_CAVERN 0xB0
+#define EVENTCHKINF_ENTERED_LAKE_HYLIA 0xB1
+#define EVENTCHKINF_ENTERED_GERUDO_VALLEY 0xB2
+#define EVENTCHKINF_ENTERED_GERUDOS_FORTRESS 0xB3
+#define EVENTCHKINF_ENTERED_LON_LON_RANCH 0xB4
+#define EVENTCHKINF_ENTERED_JABU_JABUS_BELLY 0xB5
+#define EVENTCHKINF_ENTERED_GRAVEYARD 0xB6
+#define EVENTCHKINF_ENTERED_ZORAS_FOUNTAIN 0xB7
+#define EVENTCHKINF_ENTERED_DESERT_COLOSSUS 0xB8
+#define EVENTCHKINF_ENTERED_DEATH_MOUNTAIN_CRATER 0xB9
+#define EVENTCHKINF_ENTERED_GANONS_CASTLE_EXTERIOR 0xBA
+#define EVENTCHKINF_COMPLETED_FOREST_TRIAL 0xBB
+#define EVENTCHKINF_COMPLETED_WATER_TRIAL 0xBC
+#define EVENTCHKINF_COMPLETED_SHADOW_TRIAL 0xBD
+#define EVENTCHKINF_COMPLETED_FIRE_TRIAL 0xBE
+#define EVENTCHKINF_COMPLETED_LIGHT_TRIAL 0xBF
+#define EVENTCHKINF_NABOORU_ORDERED_TO_FIGHT_BY_TWINROVA 0xC0
+#define EVENTCHKINF_SPOKE_TO_SARIA_ON_BRIDGE 0xC1
+#define EVENTCHKINF_DISPELLED_GANONS_TOWER_BARRIER 0xC3
+#define EVENTCHKINF_RETURNED_TO_TEMPLE_OF_TIME_WITH_ALL_MEDALLIONS 0xC4
+#define EVENTCHKINF_SHEIK_SPAWNED_AT_MASTER_SWORD_PEDESTAL 0xC5
```

Comment:
`EVENTCHKINF_RETURNED_TO_TEMPLE_OF_TIME_WITH_ALL_MEDALLIONS`
The cutscene starting only checks for Spirit and Shadow medallions. Even though it can be considered a check for all medallions, assuming no glitches/sequence breaks, it would still be good to keep this accurate.

It can probably be named after starting the Light Arrow Cutscene, which is what the relevant code is doing anyway

Reply:
Probably something along the lines of `EVENTCHKINF_STARTED_LIGHT_ARROW_CUTSCENE` would be better.